docs/src/man/guide.md

```diff
@@ -61,13 +61,15 @@ a single empty file at the moment, but we'll be adding to it later on.
 Add the following to your `make.jl` file
 
 ```julia
+push!(LOAD_PATH,"../src/")
```

Comment:
From [this](https://juliadocs.github.io/Documenter.jl/latest/man/guide.html#Setting-up-the-folder-structure-1) section: "For this guide we'll be using a package".